py-polars/polars/expr/datetime.py

```diff
@@ -1454,9 +1457,20 @@ def replace_time_zone(
             self._pyexpr.dt_replace_time_zone(time_zone, ambiguous._pyexpr)
         )
 
+    @deprecate_renamed_function("total_days", version="0.19.3")
```

Comment:
I usually move these deprecated functions to the bottom of the file, so they don't bother me anymore. If you want you can do the same (not required though).